themes/kairos-theme/gulpfile.js

```diff
@@ -0,0 +1,37 @@
+var elixir = require('laravel-elixir');
```

Comment:
I'm confused by the laravel elixir along with go on hugo. Please explain

Reply:
https://laravel.com/docs/5.5/mix
I used laravel elixir as a framework to help define my asset pipeline. This way I can a several *javascript* and *css/sass* files and compile to a single _.js_ and _.css_. Its easy to use and very helpful as you can see.